controllers/websocket_controller.go

```diff
@@ -4,49 +4,56 @@ import (
 	"encoding/json"
 	"fmt"
 	"gcom-backend/models"
-	esi "gcom-backend/util"
-	socketio "github.com/googollee/go-socket.io"
 	"github.com/labstack/echo/v4"
+	"github.com/zishang520/socket.io/v2/socket"
+	"gorm.io/driver/sqlite"
 	"gorm.io/gorm"
 	"time"
 )
 
 func WebsocketHandler() func(context echo.Context) error {
-	wrapper, err := esi.NewWrapper()
+	io := socket.NewServer(nil, nil)
+	db, err := gorm.Open(sqlite.Open("./db/database.db"), &gorm.Config{})
```

Comment:
As long as having two database connections won't cause any weird race-condition problems or anything of a similar matter.

Reply:
im really just crossing my fingers and hoping it doesnt, will probably need to extensively test this. another concern is that it might block the main echo thing from processing